pages/1.13/security/ent/iam-api/index.md

```diff
@@ -22,11 +22,11 @@ The API supports JSON only. You must include `application/json` as your `Content
 
 # Host name and base path
 
-The host name to use will vary depending on where your app is running.
+The host name to use will vary depending on where your program is running.
 
-* If your app runs outside of the DC/OS cluster, you should use the cluster URL. This can be obtained by launching the DC/OS web interface and copying the domain name from the browser. Alternatively, you can log in to the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL. In a production environment, this should be the path to the load balancer which sits in front of your masters.
+* If your program runs outside of the DC/OS cluster, you should use the cluster URL. This can be obtained by launching the DC/OS web interface and copying the domain name from the browser. Alternatively, you can log in to the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL. In a production environment, this should be the path to the load balancer which sits in front of your masters.
 
-* If your app runs inside of the cluster, use `master.mesos`.
+* If your program runs inside of the cluster, use `master.mesos`.
```

Comment:
Rewrite `If your program runs inside of the cluster, use master.mesos.`

with `Use master.mesos, if your program runs inside of the cluster.`